platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheEncryptionIntegrationTest.groovy

```diff
@@ -65,11 +66,10 @@ class ConfigurationCacheEncryptionIntegrationTest extends AbstractConfigurationC
         configurationCache.assertStateLoaded()
 
         where:
-        encryptionTransformation | source
-        "AES/ECB/PKCS5PADDING"   | EncryptionKind.KEYSTORE
-        "AES/CBC/PKCS5PADDING"   | EncryptionKind.KEYSTORE
-        "AES/ECB/PKCS5PADDING"   | EncryptionKind.ENV_VAR
-        "AES/CBC/PKCS5PADDING"   | EncryptionKind.ENV_VAR
+        [encryptionTransformation, source] << [
```

Comment:
There was something nice about the documentation aspect of having the algorithms explicitly listed here but the structure of `SupportedEncryptionAlgorithm` makes it clear enough. 👌

Reply:
Switched back to listing explicitly. Thats probably better for testing anyways